docs/DeveloperGuide.adoc

```diff
@@ -297,15 +299,15 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |picky eater |filter food preferences |the food I cook matches my taste
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |beginner at cooking |easily search for recipes with the ingredients I already have |use up all the food in my fridge
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |save my favourite recipes |quickly navigate to them without having to search them up again
```

Comment:
Perhaps you could describe the user - "frequent user" (users who use the same recipes over and over again